install/deploy_final_non-compute_node.md

```diff
@@ -345,7 +347,7 @@ The steps in this section load hand-off data before a later procedure reboots th
 
     See [Check for Latest Documentation](../update_product_stream/README.md#check-for-latest-documentation).
 
-1. (`ncn-m001#`) Exit the typescript and move the backup to `ncn-m001`.
+1. (`ncn-m002#`) Exit the typescript and move the backup to `ncn-m001`.
```

Comment:
This command sequence starts out on `ncn-m001`. The first exit command sends you back to `ncn-m002`, and the second exit command stops the typescript. If you're concerned that this is unclear, you could break this into substeps, so that only the first substep has the `ncn-m001#` tag, and the subsequent ones have `ncn-m002#`.

```suggestion
1. (`ncn-m001#`) Exit the typescript and move the backup to `ncn-m001`.
```